mhs/common/mhs_common/workflow/tests/test_asynchronous_reliable.py

```diff
@@ -139,7 +139,12 @@ def test_construct_workflow_with_only_inbound_params(self):
         self.mock_ebxml_request_envelope.assert_called_once_with(expected_interaction_details)
         self.mock_transmission_adaptor.make_request.assert_called_once_with(URL, HTTP_HEADERS, SERIALIZED_MESSAGE,
                                                                             raise_error_response=False)
-        self.assert_audit_log_recorded_with_message_status(log_mock, MessageStatus.OUTBOUND_MESSAGE_ACKD)
+        # self.assert_audit_log_recorded_with_message_status(log_mock, MessageStatus.OUTBOUND_MESSAGE_ACKD)
+
+        log_mock.audit.assert_called_with('0101',
+                                          'Outbound Async-Reliable outbound workflow completed. Message sent to Spine'
+                                          ' and {Acknowledgment} received.',
+                                          {'Acknowledgment': 'OUTBOUND_MESSAGE_ACKD'})
```

Comment:
Optional: you could just reference the constant on `MessageStatus` instead

Reply:
I'll leave this as it is I think for the time being, I'm generally not really a fan of using variables from production code to assert things in tests, since if that variable gets changed the test will still pass but the requirement is no longer satisfied

Reply:
I know what you mean by that, I kind-of do the same thing sometimes (and have done in at least one other place in this repo).